og.module

```diff
@@ -85,11 +88,15 @@ function og_entity_predelete(EntityInterface $entity) {
     // @todo Delete user roles.
     // @see https://github.com/amitaibu/og/issues/175
     // og_delete_user_roles_by_group($entity_type, $entity);
+
```

Comment:
```suggestion
```
----------------------------------
 90 | WARNING | [x] There must be no blank line following an inline comment
 90 | WARNING | [ ] There must be no blank line following an inline comment